### PR TITLE
Overview panel layout, stats updates

### PR DESCRIPTION
**Pull request by achen2401**
https://www.pivotaltracker.com/story/show/179860775
https://www.pivotaltracker.com/story/show/179860784

Changes include:
- In Patient Risk Overview sections: there are now three panels one for graph, one for rx summary and one for alerts
- Adding stats for Rx by drug class in the rx summary panel

Example screenshot:
OverviewPanel

### CODE REVIEW

**Review by ivan-c — APPROVED** [review]

**Review by pbugni — APPROVED** [review]

**Comment by pbugni** [comment]
As per 8-Nov stand-up, we should hold this feature until the drug-class fix is also ready to go.

File: src/components/Landing.js [hunk 1]
```diff
@@ -365,21 +368,53 @@ export default class Landing extends Component {
...
+              });
+              dataSet.push(matchItem);
+            });
+            statItem = {
+              fields : summaryFields,
+              data: dataSet
+            };
+            stats[item.title] = statItem;
+          } //end if keyMatch & matchSet
+          else if (item.keyMatch) {
```
Comment (ivan-c) [inline src/components/Landing.js:412]: If I understand correctly, you could reuse the variable you assigned earlier.
```suggestion
          else if (keyMatch) {
```

File: src/components/Landing.js [hunk 2]
```diff
@@ -365,21 +368,53 @@ export default class Landing extends Component {
...
+    let stats = {};
     let config = overviewSection.statsConfig;
     if (config) {
       let dataSource = summary[config.dataKeySource] ? summary[config.dataKeySource][config.dataKey]: null;
       let statsSource = dataSource ? dataSource : [];
       if (config.data) {
         config.data.forEach(item => {
-          let o = statsSource;
-          if (item.keyMatch) {
+          let o = statsSource, dataSet = [], statItem = {};
```
Comment (ivan-c) [inline src/components/Landing.js:378]: Minor nitpick: using several different single-character variable names in a deeply nested loop can make it difficult to understand what's going on. Longer variable names can also make find/replace easier

**Review by ivan-c — APPROVED** [review]
Looks great, merge when ready!! Minor nitpicks but no blockers

**Review by ivan-c — APPROVED** [review]

**Pull request merged**